kentik_api_library/kentik_api/synthetics/synth_tests/base.py

```diff
@@ -316,6 +316,7 @@ class SynTest(_ConfigElement):
     type: TestType = field(init=False, default=TestType.NONE)
     status: TestStatus = field(default=TestStatus.ACTIVE)
     settings: SynTestSettings = field(default_factory=SynTestSettings)
+    labels: List[str] = field(default_factory=list)
```

Comment:
Is this field included in protobuf Test?
I'm getting error while reading: 'Test' object has no attribute 'labels'